pkg/ingester/activeseries/matchers_test.go

```diff
@@ -86,6 +73,48 @@ func TestMatcher_MatchesSeries(t *testing.T) {
 	}
 }
 
+func BenchmarkMatchesSeries(b *testing.B) {
+
+	trackerCounts := []int{10, 100, 1000, 10000}
+	asms := make([]*Matchers, len(trackerCounts))
+
+	for i, matcherCount := range trackerCounts {
+		configMap := map[string]string{}
+		for j := 0; j < matcherCount; j++ {
+			configMap[strconv.Itoa(j)] = fmt.Sprintf("{grafanacloud_usage_group=~%d.*}", j)
+		}
+		config, _ := NewCustomTrackersConfig(configMap)
```

Comment:
Please `require.NoError(b, err)` here, as if this instantiation breaks, then the benchmark suddently does nothing.